src/Arriba/Tools/Arriba.TfsWorkItemCrawler/DefaultCrawler.cs

```diff
@@ -32,14 +33,17 @@ public class DefaultCrawler
         private bool Rebuild { get; set; }
 
         private IEnumerable<string> ColumnNames { get; set; }
+        private ArribaLogging _log;
```

Comment:
Where does this ArribaLogging type comes from?

Reply:
Should be ArribaLog